packages/eslint-config/base.js

```diff
@@ -0,0 +1,41 @@
+// @ts-check
+
+import eslint from '@eslint/js';
+import tseslint from 'typescript-eslint';
+import eslintPluginPrettierRecommended from 'eslint-plugin-prettier/recommended';
+
+export default tseslint.config(
+  eslint.configs.recommended,
+  ...tseslint.configs.recommended,
+  // TODO: Enable after the change to eslint9 and fix errors in a followup
```

Comment:
ticketed?

Reply:
yea ill make a bunch of tickets, i think we need to do it 1 by 1